.github/workflows/ci.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       - name: Install system dependencies
         run: |
           sudo apt-get update -y
-          sudo apt-get install python3-dev graphviz libgraphviz-dev pkg-config
+          sudo apt-get install python3-dev graphviz libgraphviz-dev pkg-config uuid-dev
```

Comment:
This is necessary to make the CI pass for the Snakemake v8 upgrade, so I would just squash the two functional commits together under the heading `refactor(snakemake): upgrade snakemake 7 to snakemake 8 (#471)` and add only the separate release commit on top.